hypothesis-python/src/hypothesis/extra/pytestplugin.py

```diff
@@ -128,12 +132,35 @@ def note_statistics(stats):
             gathered_statistics[item.nodeid] = lines
             item.hypothesis_statistics = lines
 
+        # Peturb PRNG so that we can detect e.g. everything calling seed(0)
+        random.seed(random.randrange(2 ** 32))
```

Comment:
I'm not quite sure what's going on here, but why does this help? And in order to not run into the same problem we're trying to detect would it make sense to use a `random.randrange` from some `Random` object we initialize separately and store at the module level?

Reply:
Because without it an autouse fixture that does `seed(0)` will apear as a sequence of tests where all but the first have the same PRNG state before as state after, and in that case we can't tell that anything is happening.  I've expanded the comment on this.

And yes, a module-local Random instance makes sense - done.